testnet/stacks-node/src/config.rs

```diff
@@ -1968,7 +1968,7 @@ impl Default for NodeConfig {
             marf_defer_hashing: true,
             pox_sync_sample_secs: 30,
             use_test_genesis_chainstate: None,
-            always_use_affirmation_maps: false,
+            always_use_affirmation_maps: true,
```

Comment:
What is the impact of this change? Does it impact mainnet? Is this config option only ever evaluated in testnet? Did testnet node's previously default to ignoring affirmation maps? Does this only impact tests?

Reply:
> What is the impact of this change? 

When booting a node in epoch 2.05 or earlier, we rely on the affirmation map to tell us whether or not anchor blocks exist as determined by block-commits.  We know for a fact that in all testnets and mainnet, every reward cycle either has an anchor block as determined by the block-commit history or no PoX anchor block was chosen.

This had not been the case prior to resetting primary testnet to use Bitcoin regtest.

> Is this config option only ever evaluated in testnet?

Nope, it applies on mainnet too.

> Did testnet node's previously default to ignoring affirmation maps? 

Yes, for epochs prior to 2.1.

> Does this only impact tests?

I had to set this to `true` to get my mainline testnet node booted up.